metricbeat/module/system/socket_summary/_meta/fields.yml

```diff
@@ -23,6 +23,14 @@
       description: >
         All TCP connections
       fields:
+        - name: orphan_sockets
+          type: integer
+          description: >
+            A count of all orphaned tcp sockets. Only available on Linux.
+        - name: socket_memory
```

Comment:
socket can be a bit redundant here, what about naming these directly `orphan` and `memory`?